tests/headers/opaque-template-inst-member.hpp

```diff
@@ -0,0 +1,12 @@
+// bindgen-flags: --opaque-type 'OpaqueTemplate'
+
+template<typename T>
+class OpaqueTemplate {
+    T mData;
+    bool mCannotDebug[40];
```

Comment:
I think instantiating with `T = bool` so that alignment is lower, or replace `bool mCabnnotDebug[40]` with `unsigned long long mCannotDebug[40]` should do the trick.